environs/config/config.go

```diff
@@ -423,7 +424,7 @@ var defaultConfigValues = map[string]interface{}{
 	IgnoreMachineAddresses:       false,
 	"ssl-hostname-verification":  true,
 	"proxy-ssh":                  false,
-	DefaultSpace:                 "_default",
+	DefaultSpace:                 "",
```

Comment:
In anticipation of this change being made by @manadart .  _default will not pass the names.IsValidSpace() checks.